LDH/Day4/완주하지 못한 선수.js

```diff
@@ -0,0 +1,22 @@
+function solution(participant, completion) {
+    
+    let win = {};
+    
+    for (let name of participant) {
+        if(win[name]) {
+            win[name] += 1;
+        } else {
+            win[name] = 1;
+        }
+    }
+    for (let name of completion) {
+        if(win[name]) {
+            win[name] -= 1;
+        }
+    }
```

Comment:
완주자 제외 시, 모든 참가자는 이미 win 객체에 카운트 되어있으므로 따로 win[name]이 존재하는지 확인하지 않아도 됩니다.
```suggestion
    for (let name of completion) {
            win[name]--;
    }
```